kld/src/ldk/event_handler.rs

```diff
@@ -500,14 +504,17 @@ impl EventHandler {
                     Vec::new(),
                     destination_address.script_pubkey(),
                     tx_feerate,
+                    Some(PackedLockTime(
+                        BestBlock::from_network(Network::Bitcoin).height(),
```

Comment:
This will just return the genesis block. We need the current best block height which you can get from ```bitcoind_client.get_best_block().1```.